catboost/private/libs/algo/greedy_tensor_search.cpp

```diff
@@ -1109,6 +1145,30 @@ static void MarkFeaturesAsUsed(
     }
 }
 
+static void MarkFeaturesAsUsedWithSize(
```

Comment:
very similar to MarkFeaturesAsUsed from line 1123 -- pls use TIndexedSubset<ui32> and avoid copy-paste

Reply:
deleted MarkFeaturesAsUsedWithSize function